image/templates/helm/shared/templates/_certrefresh.tpl

```diff
@@ -0,0 +1,21 @@
+{{/*
+  srox.setSecuredClusterCertRefresh
+
+  Checks if Secured Cluster certificate refresh is enabled in the current context, and sets
+  the securedClusterCertRefresh accordingly.
+
+  It should be enabled for Helm and Operator installs, and only on Secured Clusters.
+   */}}
+
+{{ define "srox.setSecuredClusterCertRefresh" }}
+
+{{ $ := index . 0 }}
+{{ $env := $._rox.env }}
```

Comment:
Maybe inline this to make the `if and` more symmetric?

Reply:
Right, `$._rox.env` was used more than once in a previous iteration.